app/locales/pl/config.js

```diff
@@ -0,0 +1,16 @@
+// Ember-I18n includes configuration for common locales. Most users
```

Comment:
It appears that ember-i18n comes with polish support by default, so this file can be removed.